akka-stream/src/main/scala/akka/stream/impl/io/InputStreamSinkStage.scala

```diff
@@ -118,14 +119,14 @@ private[stream] object InputStreamSinkStage {
     extends InputStream {
 
   var isInitialized = false
-  var isActive = true
+  val isActive = new AtomicBoolean(true)
```

Comment:
This is failing MiMa. Since it's internal / private API, if should be OK to add an exclusion.